CHANGELOG.md

```diff
@@ -4,6 +4,42 @@
 
 [1]: https://www.npmjs.com/package/nodejs-pubsub?activeTab=versions
 
+## v0.25.0
```

Comment:
doesn't this release having breaking changes?

Reply:
Maybe this was a mistake, I'm not really sure how to I should be versioning for TypeScript types. I assumed this was a feature, albeit only for TypeScript users, so I went minor.

Reply:
Oh wait, I misunderstood, the retries are breaking, you're right

Reply:
Yes!  Was just saying - we should call that out :)

Reply:
+1 to calling it out, but we're not doing major semver bumps elsewhere if that's part of the question. The "breaking"-ness of the change should be more akin to bug/unintended behavior fixing.

As to calling it out, maybe add something like "If you experience usability concerns, please file an issue at <link>."

Reply:
I made a change to include a note about it. I didn't go with the usual **BREAKING** headline, I feel like that should be reserved for surface changes. Let me know if you disagree!